v2/api/network/v1api20220401preview/structure.txt

```diff
@@ -0,0 +1,213 @@
+github.com/Azure/azure-service-operator/v2/api/network/v1api20220401preview
```

Comment:
Confused where these network files are coming from? Do you need to rebase or otherwise update the PR? I don't see any reason from `azure-arm.yaml` why they would be here.

Reply:
Good catch! I've removed these files and updated the version to `2022-09-01`. Thanks!